cmd/internal/cli/sign.go

```diff
@@ -61,14 +65,23 @@ var signSifDescIDFlag = cmdline.Flag{
 	Deprecated:   "use '--sif-id'",
 }
 
+// --key
+var signPrivateKeyFlag = cmdline.Flag{
+	ID:           "privateKeyFlag",
+	Value:        &priKeyPath,
+	DefaultValue: "",
+	Name:         "key",
+	Usage:        "path to the private key file",
+}
```

Comment:
Do we want to associate this with an env var also?

Potentially a user could set an env var to have the key used always, without needing to specify the flag, in an environment that requires that?